app/templates/components/create-options-dates.hbs

```diff
@@ -9,10 +9,4 @@
     @onCenterChange={{action (mut calendarCenter) value="moment"}}
     @onSelect={{action "daysSelected"}}
   />
-  <InlineDatepicker
-    @center={{calendarCenterNext}}
-    @selectedDays={{selectedDays}}
-    @onCenterChange={{action (mut calendarCenter) value="moment"}}
-    @onSelect={{action "daysSelected"}}
-  />
```

Comment:
No real reason, I just thought it would be cleaner. I re-added it for now and hide the second one on mobile. The back link on the second calendar doesn't work however, need to look into that. Also I think UI-wise there should be one navigation at the top to go forward or backward. Having two is a little confusing. But we can handle this in the next iteration.